web-ng/src/app/components/layer-dialog/layer-dialog.component.ts

```diff
@@ -163,14 +168,22 @@ export class LayerDialogComponent implements OnDestroy {
     const fields = this.layerService.convertFieldsListToMap(this.fields);
     const formId = this.form?.id;
     const forms = this.layerService.createForm(formId, fields);
+    let contributions: string[] = [];
+    if (this.contributorsCanAddPoint && this.contributorsCanAddPolygon) {
```

Comment:
if (this.contributorsCanAddPoints) {
  allowedFeatureTypes.push('points');
}

Same for polygons.

Reply:
updated